src/sentry/web/urls.py

```diff
@@ -399,7 +399,11 @@ def init_all_applications():
     url(r'^(?P<organization_slug>[\w_-]+)/$',
         react_page_view, name='sentry-organization-home'),
     url(r'^organizations/new/$', generic_react_page_view),
-
+    url(
+        r'^organizations/(?P<organization_slug>[\w_-]+)/$',
```

Comment:
Should we redirect to the correct react route and save the user a client side redirect?

Reply:
Good point, updated